packages/rx_bloc_cli/example/README.md

```diff
@@ -568,4 +570,5 @@ For more info please visit [widget_toolkit_pin](https://pub.dev/packages/widget_
 [patrol_pub_lnk]: https://pub.dev/packages/patrol
 [patrol_cli_pub_lnk]: https://pub.dev/packages/patrol_cli
 [patrol_native_integration_lnk]: https://patrol.leancode.pl/getting-started#integrate-with-native-side
-[rx_bloc_cli_cd_setup_lnk]: docs/continuous_delivery.md
\ No newline at end of file
+[rx_bloc_cli_cd_setup_lnk]: docs/continuous_delivery.md
+[golden_tests_for_smart_widgets_link]: docs/golden_tests.md
```

Comment:
The golden_tests.md does not exist in the https://github.com/Prime-Holding/rx_bloc/tree/develop/packages/rx_bloc_cli/example/docs

Reply:
Fixed!